config/routes.rb

```diff
@@ -9,5 +9,6 @@
 
   root to: 'posts#index'
   
-  resources :posts, only: [:index, :new, :create, :edit, :update, :destroy]
-end
+  resources :posts, only: [:index, :new, :create, :edit, :update, :destroy, :show]
```

Comment:
7つのアクションすべて指定しているので、下記が良いと思います！
```suggestion
  resources :posts
```

Reply:
おっしゃる通りです！ありがとうございます直しました！